core/appHandler.js

```diff
@@ -295,7 +295,10 @@ function listUsersAPIRating1(res) {
 
 module.exports.listUsersAPI = function (req, res) {
 	var vulnKey = 'a3_sensitive_data';
-	var securityRating = ratingState[vulnKey];
+	if(vh.vPID(req.query.securityRating))
+		securityRating == req.query.securityRating
+	else
+		securityRating = ratingState[vulnKey]
 	if (securityRating == 0 ){
```

Comment:
I think this is broken now. It looks like it is serving the insecure `listUsersAPIRating1` function, when the url has `?securityRating=1'. I am testing this while logged in as an admin. 
![image](https://user-images.githubusercontent.com/12928553/126265570-1f916644-5934-4379-b2ad-49f36e8e7da8.png)

Reply:
thanks.  see new commits

Reply:
hm this is still broken for me. I'll make a new branch off of this one and see if I can get it working.